src/components/CytoscapeLayout/CytoscapeLayout.tsx

```diff
@@ -1,11 +1,13 @@
 import * as React from 'react';
 import { ReactCytoscape } from 'react-cytoscape';
+import { Spinner } from 'patternfly-react';
 import { CytoscapeConfig } from './CytoscapeConfig';
 import * as API from '../../services/Api';
 import PropTypes from 'prop-types';
 
 type CytoscapeLayoutState = {
   elements?: any;
+  isLoading?: boolean;
```

Comment:
Wondering, if we always have `isLoading` (and is being used) why we have it as optional? (same could be said from elements, but not touched in this PR).

Reply:
I agree with @josejulio probably not optional and defaults to false;

Reply:
Why false?